README.md

```diff
@@ -329,10 +324,39 @@ _WARNING: This step requires the working OCI Registry, Cluster and Kyma CLI from
        template.yaml
      ```
 
-   Now, run the following command to create and push your module operator image to the specified registry:
+2. **Module Configuration**
+   
+   To configure the module, the file `module-config.yaml` must be adjusted. This file is located at the root of the repository.
+ 
+   The following fields are available for the configuration of the module:
+   - `name`: (Required) The name of the module.
+   - `version`: (Required) The version of the module.
+   - `channel`: (Required) The channel that should be used in the ModuleTemplate. Must be a valid Kyma state.
+   - `manifest`: (Required) The relative path to the manifest file (generated in the prerequisites step).
+   - `defaultCR`: (Optional) The relative path to a YAML file containing the default Custom Resource for the module.
+   - `resourceName`: (Optional) The name for the ModuleTemplate that will be created.
+   - `namespace`: (Optional) The namespace where the ModuleTemplate will be deployed.
+   - `security`: (Optional) The name of the security scanners configuration file.
+   - `internal`: (Optional) Determines whether the ModuleTemplate should have the internal flag or not. Type is bool.
+   - `beta`: (Optional) Determines whether the ModuleTemplate should have the beta flag or not. Type is bool.
+   - `labels`: (Optional) Additional labels for the ModuleTemplate.
+   - `annotations`: (Optional) Additional annotations for the ModuleTemplate.
+   - `customStateCheck`: (Optional) Specifies custom state checks for the module.
+
+   Here is an example configuration:
+   ```yaml
+   name: kyma-project.io/module/template-operator
+   version: v1.0.0
+   channel: regular
+   manifest: template-operator.yaml
+   ```
+   
+3. **Module Build and Push**
+   
+   Now, run the following command to create the module configured in `module-config.yaml` and push your module operator image to the specified registry:
 
    ```sh
-   kyma alpha create module --version 0.0.1 --insecure --registry op-kcp-registry.localhost:8888/unsigned
+   kyma alpha create module --insecure --registry op-kcp-registry.localhost:8888/unsigned --module-config-file module-config.yaml 
```

Comment:
The followup `To make a setup work in single-cluster mode adjust the generated ...` step is erralavent for the current setup, in v1beta2, we deprecated `spec.target`